ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt

```diff
@@ -271,6 +271,17 @@ on_game_start_after_lobby = {
 				days = 0
 			}
 		}
+		every_living_character = { ## No culture in setup
+			limit = {
+				has_culture = culture:aaa_noculture
+			}
```

Comment:
Won't this only detect no culture in **characters**? We should also check the provinces.

Reply:
Yes, but I do not know how to check it in provinces.

Reply:
Added.